eslint.config.js

```diff
@@ -0,0 +1,45 @@
+import eslint from '@eslint/js'
+import tseslint from 'typescript-eslint'
+import unusedImports from 'eslint-plugin-unused-imports'
+
+export default tseslint.config(
+  {
+    ignores: ['**/generated/*'],
+  },
+  eslint.configs.recommended,
+  tseslint.configs.recommended,
+  [
+    {
+      plugins: {
+        'unused-imports': unusedImports,
+      },
+      rules: {
+        '@typescript-eslint/no-non-null-assertion': 'error',
+        '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
Can we keep this one on?

Reply:
Done: https://github.com/cowprotocol/cow-sdk/pull/299/commits/65c5dbb2aca67b690a05111ef6ce289957058ad5